packages/theme-ui/README.md

```diff
@@ -54,7 +54,7 @@ help you build UI rooted in constraint-based design principles.
 [mdx]: https://mdxjs.com
 [styled system]: https://styled-system.com
 [gatsby]: https://gatsbyjs.org
-[theme specification]: https://system-ui.com/theme
+[theme specification]: https://system-ui.com/theme-spec
```

Comment:
Thanks! I think *this* link was correct, but the one on line 81 looks like it's a relative link, which is probably causing the issue

Reply:
Lol I totally messed this up, sorry!